ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/DefaultPrivacyController.java

```diff
@@ -414,6 +415,16 @@ public boolean isGroupAdditionTransaction(final PrivateTransaction privateTransa
             .startsWith(ADD_TO_GROUP_METHOD_SIGNATURE.toHexString());
   }
 
+  @Override
+  public boolean isGroupRemovalTransaction(final PrivateTransaction privateTransaction) {
```

Comment:
Why do we need this method here? Maybe we should add it to the `PrivateTransaction` class itself? Or to an `OnchainPrivateTransactionUtil` kind of class.